platform_versioned_docs/version-23.4.0/data/data-explorer.mdx

```diff
@@ -88,6 +88,22 @@ Data Explorer supports file uploads to your private cloud storage buckets. From
 You must configure cross-origin resource sharing (CORS) for your cloud storage provider to allow file uploads from Seqera. CORS configuration differs for each provider.
 :::
 
+## Download multiple files
+
+You can download multiple files, up to a maximum of 1,000 files, from the user interface or using a script in your cloud provider's CLI.
```

Comment:
this is not correct, the 1000 files limit doesn't apply to the "download with script" flow, only to the download through browser / UI flow

Reply:
I've updated this, thanks!